README.md

```diff
@@ -0,0 +1,12 @@
+MoviesAPI
+A simple API, which provides CRUD operations over movies model.
+
+# Before running the app:
+
+`npm install`<br/>
+`yarn install` - jest is unavailable for npm (throws an error)
```

Comment:
I'm qurious about this. According to their ddocumentation they have their release in npm registry and we use that one too.

Reply:
That's the error I get when try to install jest using the command - npm i jest:
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! Maximum call stack size exceeded

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mihailhitrov/.npm/_logs/2020-12-21T14_33_43_119Z-debug.log

Reply:
I edited the README file to use yarn install for installing depenencies.

Reply:
Here is what I get
```
/testjest$ npm i jest
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^2.1.2 (node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.2.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN testjest@1.0.0 No description
npm WARN testjest@1.0.0 No repository field.

+ jest@26.6.3
added 511 packages from 350 contributors and audited 512 packages in 10.861s

23 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```